pkgs/os-specific/linux/ffado/default.nix

```diff
@@ -49,17 +50,16 @@ mkDerivation rec {
     pkg-config
     which
     python
-    pyqt5
     wrapQtAppsHook
-  ];
+  ] ++ lib.optional withMixer pyqt5;
```

Comment:
Should `wrapQtAppsHook` also only be included when `withMixer` is enabled?

I doubt it matters much, but I figured I would ask to double check if that was desired